libraries.gradle

```diff
@@ -68,12 +68,10 @@ ext {
             slf4j_api:                 "org.slf4j:slf4j-api:1.7.5",
             slf4j_log4j:               "org.slf4j:slf4j-log4j12:1.7.5",
 
-            // jaxb task
-			jaxb:           'com.sun.xml.bind:jaxb-xjc:2.2.5',
-			jaxb2_basics:   'org.jvnet.jaxb2_commons:jaxb2-basics:0.6.3',
-			jaxb2_ant:      'org.jvnet.jaxb2_commons:jaxb2-basics-ant:0.6.3',
-			jaxb2_jaxb:     'org.jvnet.jaxb2_commons:jaxb2-basics-jaxb:2.2.4-1',
-			jaxb2_jaxb_xjc: 'org.jvnet.jaxb2_commons:jaxb2-basics-jaxb-xjc:2.2.4-1',
+            // JAXB task (using the task from JAXB Commons so we may benefit from the additional options it provides)
+            jaxb_xjc:               'org.glassfish.jaxb:jaxb-xjc:2.2.11',
+            jaxb2_commons_basics:   'org.jvnet.jaxb2_commons:jaxb2-basics:0.9.3',
+            jaxb2_commons_ant:      'org.jvnet.jaxb2_commons:jaxb2-basics-ant:0.9.3',
```

Comment:
These give me a very strange error locally...

> Could not resolve all dependencies for configuration ':java6-module:jaxb'.
   > Could not resolve org.glassfish.jaxb:jaxb-xjc:2.2.11.
     Required by:
         gradle-mixed-jdk:java6-module:1.0
      > Could not GET 'https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-xjc/2.2.11/jaxb-xjc-2.2.11.pom'.
         > peer not authenticated
   > Could not resolve org.jvnet.jaxb2_commons:jaxb2-basics:0.9.3.
     Required by:
         gradle-mixed-jdk:java6-module:1.0
      > Could not GET 'https://repo1.maven.org/maven2/org/jvnet/jaxb2_commons/jaxb2-basics/0.9.3/jaxb2-basics-0.9.3.pom'.
         > peer not authenticated
   > Could not resolve org.jvnet.jaxb2_commons:jaxb2-basics-ant:0.9.3.
     Required by:
         gradle-mixed-jdk:java6-module:1.0
      > Could not GET 'https://repo1.maven.org/maven2/org/jvnet/jaxb2_commons/jaxb2-basics-ant/0.9.3/jaxb2-basics-ant-0.9.3.pom'.
         > peer not authenticated